perception/navigator_lidar_oa/src/ConnectedComponents.h

```diff
@@ -182,10 +185,9 @@ std::vector< std::vector<int> > ConnectedComponents(OccupancyGrid &ogrid, std::v
 	}
 
 	//Re-organize obstacles, how many connections to count as an obstacle?
-	int newId = 0;
 	objects.clear();
 	for (auto ii : mapObjects)  {
-		if (ii.second.strikesPersist.size() >= 10) {
```

Comment:
Minor gripe, but be careful committing file mode changes. I think you accidentally committed changes to group permissions, which could be an issue if we ever try to edit these files from another user account